build.sbt

```diff
@@ -102,6 +102,16 @@ lazy val aws = (project in file("spade-aws"))
 lazy val examples = (project in file("spade-examples"))
   .settings(commonSettings: _*)
   .settings(noPublishSettings: _*)
+  .settings(assembly / test := {})
+  .settings(
+    assembly / assemblyMergeStrategy := {
```

Comment:
do we have to assemble examples? Can just test and run within sbt console, no?

Reply:
When testing sprinkler, I needed to have a jar for that to run.  I can take this out.

Reply:
make sense, I'm ok putting it back